src/utils/utils.ts

```diff
@@ -9,4 +11,36 @@ export const utils = {
             return acc;
         }, initialMap);
     },
+
+    /**
+     * Executes JSON-RPC response validation
+     * Copied from https://github.com/ethereum/web3.js/blob/79a165a205074cfdc14f59a61c41ba9ef5d25172/packages/web3-providers/src/validators/JsonRpcResponseValidator.js
+     */
+    isValidJsonRpcResponseOrThrow: (response: any, payload = false): boolean => {
+        if (_.isObject(response)) {
+            if ((response as any).error) {
```

Comment:
If we assign `response as any` to a variable at the top, we don't need to keep casting it everywhere.